src/app/config.ts

```diff
@@ -10,6 +10,15 @@ if (process.env.NODE_ENV === 'production') {
     explorerBitcoinUI: 'https://blockstream.info',
     tdexdBaseUrl: 'https://localhost:9000',
   };
+} else if (process.env.NODE_ENV === 'development' && process.env.REACT_APP_CHAIN === 'testnet') {
```

Comment:
why should be development only? we can eventually deploy one production built for testnet only to try

Reply:
Ha ok, will fix it